templates/explore/repo_list.tmpl

```diff
@@ -2,15 +2,15 @@
 	{{range .Repos}}
 		<div class="item">
 			<div class="ui header df ac">
-				<div class="repo-title">
+				<div class="repo-title di">
 					{{$avatar := (repoAvatar . 32 "mr-3")}}
 					{{if $avatar}}
 						{{$avatar}}
 					{{end}}
 					<a class="name" href="{{.Link}}">
-						{{if or $.PageIsExplore $.PageIsProfileStarList }}{{if .Owner}}{{.Owner.Name}} / {{end}}{{end}}{{.Name}}
-					</a>
-					<div class="labels df ac fw">
+						{{- if or $.PageIsExplore $.PageIsProfileStarList }}{{if .Owner}}{{.Owner.Name}} / {{end}}{{end}}{{.Name -}}
+					</a><!--
+					--><div class="labels dif ac fw vm">
```

Comment:
Do you really want the comment emitted here? or do you just want the template not to emit whitespace?

If the latter you may want to use: 

```
{{- /* a comment with white space trimmed from preceding and following text */ -}}
```

And more usefully you can put your reasoning why this spacing has to removed as the comment...

Reply:
Yeah I guess that makes sense.

```html
					</a>
					{{-  /* trim whitespace for inline div */ -}}
					<div class="labels dif ac fw vm">
```